src/containers/PostList/components/PostsView/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PostItem from '../PostItem';
+
+class PostsView extends Component {
+    render() {
+        const {posts} = this.props;
+        return (
+            <ul>
+                {posts.map(item => (
+                    <Link key={item.id} to={`/posts/${item.id}`}>
```

Comment:
We have some styling stuff for <link> html tag in public style.css in the public folder.